functorch/experimental/control_flow.py

```diff
@@ -1,2 +1,6 @@
-from ._cond import cond, UnsupportedAliasMutationException  # noqa: F401
+from torch._higher_order_ops.cond import (  # noqa: F401  # noqa: F401
+    cond_compiled as cond,
```

Comment:
Yeah, I can add a top-level torch.cond.

Reply:
Yes, it is good to have functorch.experimental.control_flow to avoid excessive breakages.

will there be a PR for adding torch.cond?

Reply:
I think the best case is to have the torch.cond landed after SymBool PR. But the SymBool one seems difficult to land today. It might be possible over the weekend.